### DISCUSSION TITLE
What's the recommanded cuda version for v23?

### DISCUSSION THREAD

**Discussion by yslan** [Nek5000/nekRS#500] [Compiling] [answered]
I think we have atomic operations in v23 so the older CUDA is no longer supported?

**Comment by stgeke** [comment, marked as answer]
For nekRS there should be no restrictions i.e. all recent CUDA versions e.g. 10.x, 11.x and 12.x should work. The latest version (12.x) is not supported by HYPRE (only relevant if you run boomerAMG on device) and potentially AMGx but I haven't test it lately.
